Enabled AV1 decode by default.

jeffqjiangNew opened the pull request.

jeffqjiangNew committed:
* rocDecode/AV1: Enabled AV1 decode by default.

AryanSalmanpour commented on src/parser/rocparser_api.cpp:
```diff
@@ -34,11 +34,10 @@ rocDecCreateVideoParser(RocdecVideoParser *parser_handle, RocdecParserParams *pa
         parser_params->codec_type != rocDecVideoCodec_AVC &&
-        (parser_params->codec_type == rocDecVideoCodec_AV1 && (is_av1_enabled == nullptr || std::string(is_av1_enabled) != "1"))) {
-        ERR("The current version of rocDecode officially supports only the H.265 (HEVC) and H.264 (AVC) codec.");
+        (parser_params->codec_type != rocDecVideoCodec_AV1)) {
```
please remove the extra parentheses around` (parser_params->codec_type != rocDecVideoCodec_AV1)` -> `parser_params->codec_type != rocDecVideoCodec_AV1`

Also, in the main CMakeLists.txt [here](https://github.com/ROCm/rocDecode/blob/develop/CMakeLists.txt#L26) bump the rocDecode version to `0.7.0`

in the main [README.md](https://github.com/ROCm/rocDecode/blob/develop/README.md) , in the Supported Codecs section add a bullet for AV1 - 8 bit, and 10 bit

jeffqjiangNew replied:
Thanks for the reminders. Done.

jeffqjiangNew committed:
* rocDecode/AV1: Added changes suggested in code review.
 - Bumped up rocDecode version to 0.7.0.
 - Added AV1 support statement in readme file.
